src/Definition/Doc.elm

```diff
@@ -435,7 +439,12 @@ view refToMsg toggleFoldMsg docFoldToggles document =
                         )
 
                 Span docs ->
-                    span [] (List.map viewAtCurrentSectionLevel (mergeWords docs))
+                    case docs of
+                        [ d ] ->
+                            viewAtCurrentSectionLevel d
+
+                        ds ->
+                            span [ class "span" ] (List.map viewAtCurrentSectionLevel (mergeWords ds))
```

Comment:
Sometimes there's just a span around a single element and it's not really doing much. In the case of tooltips there was a span around a section and it was making styling that very specific case annoying.